frontend/src/entity/Change.tsx

```diff
@@ -0,0 +1,6 @@
+export interface Change {
+  id: string;
+  title: string;
+  summaryMarkdown: string;
```

Comment:
I think this field is optional: https://github.com/short-d/short/blob/master/backend/app/adapter/gqlapi/schema.go#L31